uw-frame-components/portal/main/partials/header.html

```diff
@@ -18,7 +18,7 @@
                          class="hidden-xs crest"
                          alt="{{portal.theme.crestalt}}">
                     <span class='title'>{{portal.theme.title}}</span>
-                    <!-- <span class="beta-logo">{{portal.theme.subtitle}}</span> -->
+                    <span class="beta-logo">{{portal.theme.subtitle}}</span>
```

Comment:
Yeah, forgot i did that. Not sure why that ever got commented out. We have the ability, thought it should be known.

![http://goo.gl/XvTd0X](http://goo.gl/XvTd0X)